src/org/rascalmpl/library/lang/rascal/tests/concrete/AtColumnRequirement.rsc

```diff
@@ -0,0 +1,36 @@
+module lang::rascal::tests::concrete::AtColumnRequirement
+
+import ParseTree;
+import Exception;
+
+lexical C = C C C | "a"@2 | "b";
+
+test bool testA() {
+    try {
+        parse(#C, "a");
+    } catch ParseError(e): {
```

Comment:
I reckon this is a common pattern in parser tests. Is there any utility function that we can use here instead?

Something like `return expectError(#C, "a");`

Reply:
You could use
```
@expected{ParseError}
test bool testA() {
    //...
}
```

Reply:
Of course! I think that's more readable and less error-prone.

Reply:
This pattern is more generic than just for parser tests. The cleanest solution I see is to extend the Rascal test support, for instance by adding an annotation for test functions (like junit does for Java), or extending the `test` keyword to take an optional exception-matching argument.
Both of these are well beyond the scope of this PR of course.

I did refactor the common code from these tests into a separate function so the tests are more readable now.